README.md

```diff
@@ -33,42 +35,38 @@ Usage
 1. Navigate to the same directory as the `Cargo.toml` you want to analyze.
 2. `cargo geiger`
 
-
 Output example
 --------------
 
 ![Example output](https://user-images.githubusercontent.com/3704611/53132247-845f7080-356f-11e9-9c76-a9498d4a744b.png)
 
-
 Why even care about unsafe Rust usage?
 --------------------------------------
 
-When and why to use unsafe Rust is out of scope for this project, it is simply
+When and why to use unsafe Rust is out of scope for this project; it is simply
 a tool that provides information to aid auditing and hopefully to guide
-dependency selection. It is however the opinion of the author of this project
+dependency selection. It is, however, the opinion of the author of this project
 that __libraries choosing to abstain from unsafe Rust usage when possible should
 be promoted__.
 
 This project is an attempt to create pressure against __unnecessary__ usage of
 unsafe Rust in public Rust libraries.
 
-
 Why the name?
 -------------
 
 <https://en.wikipedia.org/wiki/Geiger_counter>
 
-Unsafe code and ionizing radiation have something in common, they are both
-inevitable in some situations and both should preferably be safely contained!
+Unsafe code and ionizing radiation have something in common, and they are both
+inevitable in some situations, and should preferably be safely contained!
```

Comment:
The first added "and" seems out of place when I read the new version as well as the added comma before the last "and". The part after the first comma is means to explain that that have in common. Does that make sense?

Reply:
Unsafe code, like ionizing radiation, is unavoidable in some situations and should be safely contained!

Reply:
> Maybe this is too terse? I can just revert this change all together if you want. It just seemed verbose.
>
>> Unsafe code, like ionizing radiation, is unavoidable in some situations and should be safely contained!

Perfect! Let's use that one.

Reply:
Merging now and will edit in your suggestion after the merge.